esiclient/utils.py

```diff
@@ -39,7 +39,7 @@ def get_network_info_from_port(port, client):
     return get_network_display_name(network), fixed_ip
 
 
-def get_full_network_info_from_port(port, client):
+def get_full_network_info_from_port(port, client, get_trunk_details=True):
```

Comment:
I think this is better defaulted as "False", as that's the more usual default case.

Reply:
Fixed.